collector/cluster-insight-controller.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
The short response is: no official comments in JSON.

The long response is:
JSON syntex does not allow any comments. I found a long discussion how to add comments to JSON, and many people say that comments should never be included in JSON because it is just data.